test/multiverse/suites/sidekiq/sidekiq_instrumentation_test.rb

```diff
@@ -77,7 +77,11 @@ def test_delayed_with_malformed_yaml
       YAML.stubs(:load).raises(RuntimeError.new("Ouch"))
       run_delayed
       assert_metric_and_call_count(ROLLUP_METRIC, JOB_COUNT)
-      assert_metric_and_call_count(DELAYED_FAILED_TXN_NAME, JOB_COUNT)
+      if RUBY_VERSION >= '3.0.0'
+        assert_metric_and_call_count(DELAYED_TRANSACTION_NAME, JOB_COUNT)
+      else
+        assert_metric_and_call_count(DELAYED_FAILED_TXN_NAME, JOB_COUNT)
+      end
```

Comment:
Psych's `#unsafe_load` method for YAML causes Ruby 3+ to return the correct transaction name rather than the failed transaction name. This PR adds 4.2.0, a now-deprecated version of Sidekiq, to the Envfile. This test is only run on versions under 5.0.0, so I believe this is why we missed this test issue when we initially added `#unsafe_load` support to our Sidekiq instrumentation.